Source: https://fedorahosted.org/pipermail/pam-developers/2012-July/000287.html
Upstream: no
From: Stevan Bajić <stevan at bajic.ch>

RLIMIT_* variables are no longer defined unless you explicitly include
sys/resource.h.

Signed-off-by: Diego Elio Pettenò <flameeyes at flameeyes.eu>
---
 modules/pam_unix/pam_unix_acct.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/modules/pam_unix/pam_unix_acct.c b/modules/pam_unix/pam_unix_acct.c
index 8e90cc9..4a362f8 100644
--- a/modules/pam_unix/pam_unix_acct.c
+++ b/modules/pam_unix/pam_unix_acct.c
@@ -41,6 +41,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <sys/resource.h>
 #include <syslog.h>
 #include <pwd.h>
 #include <shadow.h>
-- 
1.7.8.6

